students/gyeongjae-ham/westagram/users/views.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from django.http     import JsonResponse
+from django.views    import View
+
+from .models import User
+
+class CreateUserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if User.objects.filter(email=data['email']).exists() == True:
+                return JsonResponse({'MESSAGE':'이미 등록된 이메일입니다.'}, status=400)
```

Comment:
리턴 메시지는 클라인언트가 보는 것이 아니고 프론트가 보는 메시지 입니다. 그래서 굳이 한글로 친절하게 안 보내셔도 됩니다.
다음 메시지가 일반적입니다.
```suggestion
                return JsonResponse({'MESSAGE':'DUPLICATED_EMAIL'}, status=400)
```